python/cudf/cudf/core/column/lists.py

```diff
@@ -285,3 +285,36 @@ def take(self, lists_indices):
             raise
         else:
             return res
+
+    def unique(self):
+        """
+        Returns unique values for each list in the column, order for each
+        unique value is not gaurenteed.
+
+        Returns
+        -------
+        ListColumn
+
+        Examples
+        --------
+        >>> s = cudf.Series([[1, 1, 2, None, None], None, [4, 4], []])
+        >>> s
+        0    [1.0, 1.0, 2.0, nan, nan]
+        1                         None
+        2                   [4.0, 4.0]
+        3                           []
+        dtype: list
+        >>> s.list.unique() # Order of list elements is not gaurenteed
+        0              [1.0, 2.0, nan]
```

Comment:
According to my new fix (https://github.com/rapidsai/cudf/pull/7662), `NaN` will be treated as different values. Thus, the result will be `[1.0, 2.0, nan, nan]`.

Reply:
Also, since the PR #7662 is not merged yet, this PR should wait to be merged after it. Thanks.

Reply:
`pandas.Series.unique` handles deduplicating `nan` and `<NA>` values. We should at least note the difference in behavior here, but we should probably raise an issue for libcudf to have an option for `null` / `nan` handling as that would be the expected Pandas behavior.

Reply:
Good point. I'm thinking about how to change the API to allow handling the discrepancy between Pandas and Spark.